cpanfile

```diff
@@ -23,6 +33,6 @@ requires 'Test::Portability::Files';
 requires 'Test::RequiresInternet';
 requires 'Test::Spelling';
 requires 'Test::Synopsis';
-requires 'Test::Vars';
+#requires 'Test::Vars'; # requires Perl 5.010000
```

Comment:
I think we can change this to: `require 'Test::Vars' if $] ge '5.01';`

Reply:
better to use `>= '5.10'`

Reply:
Sure! I just lifted my code from https://perldoc.pl/perlvar#$OLD_PERL_VERSION :)

Reply:
This was just recently updated in blead
https://github.com/Perl/perl5/commit/601390ce200db31ae8eecba5649a8c1f2e14ca3d

Reply:
Thanks for pointing that out!